docs/make_work.jl

```diff
@@ -76,7 +80,7 @@ function doit(strict::Bool = true, local_build::Bool = false)
 
   # remove the copied documentation agains
   for pkg in other_packages
```

Comment:
`# remove the copied documentation again`
`for (pkg,pkgdir) in other_packages`

Reply:
Ouch, indeed, I forgot to commit that (hence it worked locally... *sigh*). Should be fixed now.

Reply:
Yes, thanks.
Now I am just irritated why a line in `build_doc` is claimed to be not covered by tests. I thought that either the function is not called at all (then the whole function should be marked as not covered) or this line must be reached.